Controllers/TimeWorkedAndVacationController.cs

```diff
@@ -105,7 +102,9 @@ public async Task<IActionResult> Edit(int? id)
         }
 
         // POST: TimeWorkedAndVacation/Edit/5
-        // protect overposting attacks, enable the specific properties you want to bind to, for 
+        // To protect from overposting attacks, enable the specific properties you want to bind to, for 
+        // more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
+        [HttpPost]
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> Edit(int id, [Bind("TimeWorkedAndVacationId,TotalNumberOfDaysWorked,TotalNumberOfAvailableVacationDays")] TimeWorkedAndVacation timeWorkedAndVacation)
```

Comment:
Again here buddy. Remember your datacontext literally wants you to give it a new object, and updated object, or an object to remove. You need something in between it and your controller to create those objects, update those objects.